### ISSUE TITLE
Wadgen fails if run on a ROM in the current directory when run over SSH

### ISSUE THREAD

**Issue by ProfessorKaos64** [svkaiser/Doom64EX#32]
With doom.z64 in PWD:

```
doom64ex -wadgen doom64.z64 
...
Writing IWAD File...
ERROR: File_Open: Couldn't create doom64.z64/doom64.wad
```

In another directory:

```
mkdir output_wad && cd output_wad
 doom64ex -wadgen ../doom64.z64 
```

Probably something to do with XDG_DATA_HOME not being set on my VPS. It may be nice to have a fallback if this scenario occurs. Just something I noticed.

**Comment by dotfloat** [comment]
Thanks. I intend to start refactoring Wadgen after I get done setting up nightly builds. (Shouldn't take too long.)

**Comment by ProfessorKaos64** [comment]
Np, just checking, since I was unsure of the intricate behavior here. Thanks for looking at this.

**Comment by dotfloat** [comment]
You mean the one about clang 3.6? Yeah. I'm currently in the process of setting up a travis-ci config. They run Ubuntu 12.04 (precise) build VMs, so that should inform us of what the minimum versions of gcc and clang are.

**Comment by dotfloat** [comment]
So it seems that GCC 5 and Clang 3.5 with libstdc++-5 (usually installed alongside GCC 5) are the minimum compiler versions needed to compile this thing.

**Comment by ProfessorKaos64** [comment]
You can specify a CXX flagset like `-DCMAKE_CXX_FLAGS="-O2 -g -stdlib=libc++` to use that.

**Comment by dotfloat** [comment]
Yes, I know. But I didn't manage to install libc++ in either my 12.04 VM or travis (which is also 12.04). Seems like the llvm repo doesn't have it or something.

The packages in question are called: `libc++-dev` and `libc++abi-dev`.

**Comment by dotfloat** [comment]
The current wadgen uses the standard `fopen`/`fwrite`/`fclose` functions instead of POSIX and puts the files into `$XDG_DATA_DIR/doom64ex` instead of wherever it wants.

Hopefully this solves the problem in OP.
